app/config/releases.exs

```diff
@@ -105,7 +105,9 @@ config :meadow,
       bucket: aws_secret("meadow", dig: ["buckets", "sitemap"]),
       path: "/"
     ],
-    sitemap_url: aws_secret("meadow", dig: ["dc", "base_url"])
+    base_url: aws_secret("meadow", dig: ["dc", "base_url"]),
+    sitemap_url:
+      aws_secret("meadow", dig: ["dc", "base_url"], apply: &{:ok, Path.join(&1, "api/sitemap")})
```

Comment:
@mbklein Seems like this did the trick, at least logically. How long until we know if Google and others like it?